vsts/node-release.yaml

```diff
@@ -0,0 +1,185 @@
+name: Release_$(BuildID)_$(Date:yyyyMMdd)$(Rev:.r)
+resources:
+- repo: self
+  clean: true
+
+phases:
+
+- phase: Phase_1
+  displayName: Create Release Branch
+
+  condition: succeededOrFailed()
+  queue:
+    name: Hosted
```

Comment:
>Hosted [](start = 10, length = 6)

I'd be curious to test if this could run on Linux (using powershell core?) npm & node operations are notably faster on linux.